bravado/requests_client.py

```diff
@@ -266,7 +266,7 @@ def result(self, timeout=None):
         response = self.session.send(
             prepared_request,
             timeout=self.build_timeout(timeout),
-            allow_redirects=self.misc_options['allow_redirects'],
+            follow_redirects=self.misc_options['follow_redirects'],
```

Comment:
I don't have an objection to it being renamed, but it's worth pointing out that it's called this becaise that's the name for it in `requests` itself. That's also the reason why the tests have started failing. You can fix them by changing the parameter name here back to `allow_redirects` if you want to pursue the rename.